frontend/dispatcher.lua

```diff
@@ -48,6 +48,7 @@ local settingsList = {
     stats_calendar_view = { category="none", event="ShowCalendarView", title=_("Statistics calendar view"), device=true, separator=true,},
     history = { category="none", event="ShowHist", title=_("History"), device=true,},
     open_previous_document = { category="none", event="OpenLastDoc", title=_("Open previous document"), device=true,},
+    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = 'Texteditor: re-open last edited file', device = true, },
```

Comment:
please use `gettext` _() for the title so it can be translated

Reply:
I think the text can probably be shortened a little without losing any clarity.
```suggestion
    edit_last_edited_file = { category = "none", event = "OpenLastEditedFile", title = _("Text editor: open last file"), device = true, },
```